src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsUcCaseData.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SscsUcCaseData {
+
+    private List<String> ucWriteFinalDecisionPhysicalDisabilitiesQuestion;
+    private List<String> ucWriteFinalDecisionMentalAssessmentQuestion;
+    private String ucWriteFinalDecisionMobilisingUnaidedQuestion;
+    private String ucWriteFinalDecisionStandingAndSittingQuestion;
+    private String ucWriteFinalDecisionReachingQuestion;
+    private String ucWriteFinalDecisionPickingUpQuestion;
+    private String ucWriteFinalDecisionManualDexterityQuestion;
+    private String ucWriteFinalDecisionMakingSelfUnderstoodQuestion;
+    private String ucWriteFinalDecisionCommunicationQuestion;
+    private String ucWriteFinalDecisionNavigationQuestion;
+    private String ucWriteFinalDecisionLossOfControlQuestion;
+    private String ucWriteFinalDecisionConsciousnessQuestion;
+    private String ucWriteFinalDecisionLearningTasksQuestion;
+    private String ucWriteFinalDecisionAwarenessOfHazardsQuestion;
+    private String ucWriteFinalDecisionPersonalActionQuestion;
+    private String ucWriteFinalDecisionCopingWithChangeQuestion;
+    private String ucWriteFinalDecisionGettingAboutQuestion;
+    private String ucWriteFinalDecisionSocialEngagementQuestion;
+    private String ucWriteFinalDecisionAppropriatenessOfBehaviourQuestion;
+    private String ucWriteFinalDecisionSchedule7ActivitiesApply;
+    private List<String> ucWriteFinalDecisionSchedule7ActivitiesQuestion;
+    private YesNo doesSchedule8Paragraph4Apply;
+    private YesNo showSchedule8Paragraph4Page;
+    private YesNo showSchedule7ActivitiesPage;
+    private YesNo doesSchedule9Paragraph4Apply;
```

Comment:
The field lcwaAppeal is missing

Reply:
Probably it should be a YesNo field, instead of a String.

Reply:
I've used the same type of field as for ESA for consistency

Reply:
the tribunals code translates the boolean to a YesNo field, wouldn't it be better if the translation was not necessary, if the type here is correct?

Reply:
I agree- but then we really should go back and back the wca field for ESA a Yes/No field for consistency.   But where do we stop... what about the generate notice field - and the PIP fields, or adjourn case fields.

Reply:
Go with consistency and then we can clean up when we've finished the decision notice work to make everything Yes/NO across the board, if we think it's worthwhile. Alternatively, feel free to make the change @ryandac and we can approve.